python/compiler/aiecc/cl_arguments.py

```diff
@@ -270,6 +270,24 @@ def parse_args(args=None):
         const=True,
         help="Generate control packets for configuration",
     )
+    parser.add_argument(
+        "--ctrlpkt-name",
+        dest="ctrlpkt_name",
+        default="{0}_ctrlpkt.bin",
+        help="Output filename for control packet binary data. `{0}` is replaced with the device name.",
+    )
+    parser.add_argument(
+        "--ctrlpkt-dma-seq-name",
+        dest="ctrlpkt_dma_seq_name",
+        default="{0}_ctrlpkt_dma_seq.bin",
+        help="Output filename for control packet DMA sequence. `{0}` is replaced with the device name.",
+    )
+    parser.add_argument(
+        "--ctrlpkt-elf-name",
+        dest="ctrlpkt_elf_name",
+        default="{0}_ctrlpkt.elf",
+        help="Output filename for control packet DMA sequence and control packet binary data in combined ELF file. `{0}` is replaced with the device name.",
+    )
```

Comment:
Why the new elf name option? One already exists. I don't think it need differentiation based on what's inside, esp in the future.